pkg/api/const.go

```diff
@@ -223,6 +223,10 @@ const (
 	StandardVMType = "standard"
 	// DefaultRunUnattendedUpgradesOnBootstrap sets the default configuration for running a blocking unattended-upgrade on Linux VMs as part of CSE
 	DefaultRunUnattendedUpgradesOnBootstrap = true
+	// DefaultEnableUnattendedUpgrades sets the default configuration for running unattended-upgrade on a regular schedule in the background
+	DefaultEnableUnattendedUpgrades = true
+	// DefaultEnableUnattendedUpgradesAzureStack sets the default configuration for running unattended-upgrade on a regular schedule in the background for Azure Stack Hub
+	DefaultEnableUnattendedUpgradesAzureStack = true
```

Comment:
Could you please change this default to `false`?

Reply:
I think the question of the default is what is least surprising.  Today, the default is True (in every aks-engine released since the start)

To change the default would be a breaking change - to warn about it first is better and let people actively add the setting to what they want it to be.

Reply:
I think it's @jadarsie's call for Azure Stack Hub, though Michael brings up a relevant point about the fact that such a change would be breaking existing behaviors.

Reply:
I think you are right @Michael-Sinz, better to keep the existing behavior.